packages/graphql/src/graphql.ts

```diff
@@ -79,14 +60,14 @@ type InternalData = {
 }
 
 function typeToGraphQLOutputTypeInternal(type: types.Type, internalData: InternalData): GraphQLOutputType {
-  const { inspectedTypes, knownTypes, defaultName } = internalData
+  const { inspectedTypes, knownOutputTypes, defaultName } = internalData
   // If the type has already been explored, then return the output type that has
   // already been generated
   if (inspectedTypes.has(type)) {
     // ⚠️ Possible pain point: `typeToGraphQLTypeInternal` relies on the fact
     // that _every single type_ that appears in `inspectedTypes` must also have
     // an associated generated type here
-    return knownTypes.get(type)!!
+    return knownOutputTypes.get(type)!!
```

Comment:
I think that this is the source of the current bug where some GraphQL values are `undefined` (it may be because the invariant is broken now that there are two `knownTypes` structures!)